src/workflows-app/fixtures/featured-workflows.ts

```diff
@@ -11,6 +11,81 @@ export type FeaturedWorkflowSet = Omit<WorkflowMethodSet, 'methods'> & { methods
 
 export type FeaturedWorkflow = FeaturedWorkflowMethod | FeaturedWorkflowSet;
 
+export const featuredWarpWorkflows = [
```

Comment:
Is there a reason to separate these from the `featuredWorkflowsData`?

Reply:
I was trying to future proof these, in case they change in the future, but also `featuredWorkflowsData` includes input/output mappings that make the list really long :joy: I can remove if necessary